### ISSUE TITLE
Cannot generate Windows.Win32.UI.Accessibility.IUIAutomation

### ISSUE THREAD

**Issue by mcooley** [microsoft/CsWin32#736]
## Actual behavior
error PInvoke000: An internal error occurred: Failed to generate IUIAutomation. Failed while generating the method: IntSafeArrayToNativeArray. Specified argument was out of the range of valid values.

## Expected behavior
CsWin32 can generate a projection for IUIAutomation.

## Repro steps

1. `NativeMethods.txt` content:
Windows.Win32.UI.Accessibility

2. `NativeMethods.json` content (if present):
N/A (used defaults)

3. Any of your own code that should be shared?
N/A

### Context
- CsWin32 version: 0.2.98-beta
- Win32Metadata version (if explicitly set by project): 36.0.9-preview
- Target Framework: net6.0

**Comment by AArnott** [comment]
Evidently this repro requires updating to metadata newer than CsWin32 is tested with. I'm working on a metadata upgrade to accommodate this.
